QuoteGeneration/quote_wrapper/tdx_quote/td_ql_logic.cpp

```diff
@@ -1154,6 +1154,20 @@ tee_att_error_t tee_att_config_t::ecdsa_init_quote(sgx_ql_cert_key_type_t certif
         }
         SE_TRACE(SE_TRACE_DEBUG, "Successfully verified ECDSA Blob.\n");
         p_qe_target_info->mr_enclave = tdqe_report_body.mr_enclave;
+
+        if (NULL == m_tdqe_report_body) {
+            m_tdqe_report_body = (sgx_report_body_t*)malloc(sizeof(sgx_report_body_t));
```

Comment:
On error cases, it is not clear to me that the memory is freed?

Reply:
The `m_tdqe_report_body` is a member variable, it will be freed in the destructor.